services/workflows-service/src/business-report/business-report.controller.internal.ts

```diff
@@ -55,6 +56,26 @@ export class BusinessReportControllerInternal {
     }: CreateBusinessReportDto,
     @CurrentProject() currentProjectId: TProjectId,
   ) {
+    const customer = await this.customerService.getByProjectId(currentProjectId);
+    const maxBusinessReports = customer.config?.maxBusinessReports;
+
+    if (isNumber(maxBusinessReports) && maxBusinessReports > 0) {
+      const businessReportsCount = await this.businessReportService.count(
+        {
+          where: {
```

Comment:
Is the `where` necessary? It counts the number of existing reports for a specific business, not the number of BusinessReports the customer has

Reply:
🤦
Thanks for the catch
Copilot autocompleted it and I didn't review it